astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
My motivation for changing to `min` / `max` was to shorten it.
So -1 on something longer and making the user type "airmass" several times:
```python
airmass_constraint = AirmassConstraint(airmass_max=1.42)
```
I think built-in name collissions like `min` or `max` or `format` are OK in short methods.
If you want to avoid them, `low` and `high` or `lo` / `hi` is another option.